dherault/serverless-offline
# Allowing provided runtime to work.

This should fix #570 

How to test:

YML:
```
service: localTest # NOTE: update this with your service name
plugins:
  - serverless-offline
provider:
  name: aws
  runtime: provided
functions:
  hello:
    handler: index.php
    layers:
      - arn:aws:lambda:us-east-1:209497400698:layer:php-72-fpm:6
    events:
      - http: ANY /
      - http: 'ANY {proxy+}'

```

index.php:

Can be anything for example:
`Hello <?php echo $_GET['name'] ?? "World";?>
`


sls offline will work now.

This is a first approach.
I see lots of ways to improve like reusing the same container and not having to rebuild each and every request.
Comments and tests with another layers are always welcome.
